### PR TITLE
feat(hpc-vmware-managed-vcd): cleanup tests with test-utils

### PR DESCRIPTION
**Pull request by chipp972**
ref: MANAGER-15997

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/vcd-veeam-postga` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-15997 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/general-informations/DatacentreGeneralInformation.spec.tsx [hunk 1]
```diff
@@ -5,16 +5,16 @@ import {
...
+import {
   labels,
   mockEditInputValue,
   renderTest,
-} from '../../../../test-utils';
-import {
-  DEFAULT_TIMEOUT,
   mockSubmitNewValue,
-} from '../../../../test-utils/uiTestHelpers';
+} from '../../../../test-utils';
```
Comment (rjamet-ovh) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/general-informations/DatacentreGeneralInformation.spec.tsx:17]: tsconfig path does not work on this folder? e.g. `@/test-utils`

File: packages/manager/apps/hpc-vmware-managed-vcd/vitest.config.js [hunk 2]
```diff
@@ -18,6 +18,9 @@ export default defineConfig({
         'src/App.tsx',
         'src/index.tsx',
         'src/tracking.constant.ts',
+        '404.tsx',
+        'src/**/*.spec.ts',
+        'src/**/*.spec.tsx',
```
Comment (rjamet-ovh) [inline packages/manager/apps/hpc-vmware-managed-vcd/vitest.config.js:23]: Why exclude all the spec files ?

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/datacentre-general-information-tile/DatacentreGeneralInformationTile.spec.tsx [hunk 3]
```diff
@@ -78,8 +79,8 @@ describe.skip('DatacentreGeneralInformationTile component unit test suite', () =
     const { getByText } = render(
       <QueryClientProvider client={queryClient}>
         <DatacentreGeneralInformationTile
-          vcdOrganization={vcdOrg}
-          vcdDatacentre={datacentre}
+          vcdOrganization={vcdOrg as VCDOrganization}
+          vcdDatacentre={datacentre as VCDDatacentre}
```
Comment (tibs245) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/datacentre-general-information-tile/DatacentreGeneralInformationTile.spec.tsx:83]: Maybe declare the constante on declaration with this type ?

File: packages/manager/core/test-utils/src/utils/ui-test-helpers-ods17.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { ODS_ICON_NAME } from '@ovhcloud/ods-components';
-import { screen, waitFor, fireEvent } from '@testing-library/react';
+import { within, screen, waitFor, fireEvent } from '@testing-library/react';
 import '@testing-library/jest-dom';
 
 export const waitForOptions = {
```
Comment (tibs245) [inline packages/manager/core/test-utils/src/utils/ui-test-helpers-ods17.ts:5]:
```suggestion
export const WAIT_FOR_DEFAULT_OPTIONS = {
```

File: packages/manager/core/test-utils/src/utils/ui-test-helpers-ods17.ts [hunk 5]
```diff
@@ -20,20 +20,41 @@ export const checkModal = async ({
       : expect(modal).not.toBeInTheDocument();
   }, waitForOptions);
 
+export const checkModalText = ({
```
Comment (tibs245) [inline packages/manager/core/test-utils/src/utils/ui-test-helpers-ods17.ts:23]:
```suggestion
export const assertModalText = ({
```

**Review by tibs245 — APPROVED** [review]
Just optionnal nomenclature suggestion. But good utils function.

File: packages/manager/apps/hpc-vmware-managed-vcd/vitest.config.js (hunk 2 above)
Comment (chipp972) [reply packages/manager/apps/hpc-vmware-managed-vcd/vitest.config.js:23]: we don't test the tests files so they should not be included in the coverage report

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/datacentre-general-information-tile/DatacentreGeneralInformationTile.spec.tsx (hunk 3 above)
Comment (chipp972) [reply packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/datacentre-general-information-tile/DatacentreGeneralInformationTile.spec.tsx:83]: They are mocked so they don't correspond exactly to the types probably
No need to include a full typed mock here but we don't want the warning either

File: packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/general-informations/DatacentreGeneralInformation.spec.tsx (hunk 1 above)
Comment (chipp972) [reply packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/general-informations/DatacentreGeneralInformation.spec.tsx:17]: my IDE displays an error but the test runs and succeed so I will change it

**Pull request merged**